velociraptor/tools/lines.py

```diff
@@ -83,13 +100,25 @@ def binned_mean_line(
 
             centers.append(0.5 * (x_bins[bin - 1].value + x_bins[bin].value))
 
+        # If the number of data points in the bin is less than minimum_in_bin,
+        # collect these data points if needed
+        elif (minimum_in_bin > indicies_in_this_bin.sum() > 0) and return_additional:
+            for x_point, y_point in zip(x[indicies_in_this_bin].value, y[indicies_in_this_bin].value):
+                additional_x.append(x_point)
+                additional_y.append(y_point)
+
     means = unyt.unyt_array(means, units=y.units, name=y.name)
     standard_deviations = unyt.unyt_array(
         standard_deviations, units=y.units, name=f"{y.name} ($sigma$)"
     )
     centers = unyt.unyt_array(centers, units=x.units, name=x.name)
 
-    return centers, means, standard_deviations
+    if not return_additional:
+        return centers, means, standard_deviations
+    else:
+        additional_x = unyt.unyt_array(additional_x, units=x.units, name=x.name)
```

Comment:
I would like this to be performed outside of the if/else above  with all of the other conversions to `unyt_array`. It seems cleaner that way.